lib/index.js

```diff
@@ -89,6 +89,42 @@ function objectMerge(obj1, obj2) {
 }
 
 /**
+ * Yields a warning for a given deprecated property.
+ * @function
+ * @param {string} propertyName - The property to warn about.
+ */
+function deprecatedPropertyWarning(propertyName) {
+  if (propertyName === 'tag') {
+    console.warn('tag will be deprecated in v1.0.0');
```

Comment:
Done. I thought there is something breaking.
Let me know if anything else is to be handled?